mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -70,6 +149,27 @@ AUI.add(
 							todoList.append(taskHtml);
 
 							taskInput.val('');
+
+							var remainingDiv = A.one('.remaining-tasks-count');
+
+							var profileTasksCount = A.one('.profile-tasks-count');
+
+							var taskList = A.one('.task-list');
+
+							var taskListLi = taskList.all('li');
+
+							var remainingCount = taskListLi.size();
+
+							if (remainingCount) {
+								remainingDiv.html(remainingCount);
+
+								profileTasksCount.html(remainingCount);
+							}
+							else if (remainingCount === 0) {
+								remainingDiv.html('0');
+
+								profileTasksCount.html('0');
+							}
```

Comment:
There is code duplication in blocks 157-172 and 68-83. Abstract the code to a function, as previously requested in https://github.com/markocikos/liferay-internship/pull/138#discussion_r121018442